pkgs/data/fonts/oxygenfonts/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  pname = "oxygenfonts";
+  version = "unstable-20140213";
+in
+
+  stdenv.mkDerivation rec {
+    name = "${pname}-${version}";
```

Comment:
Can simply have `name = "oxygenfonts-20140213"` here and remove the `pname` and `version` variables.